stable/atlantis/README.md

```diff
@@ -46,8 +45,8 @@ The following options are supported.  See [values.yaml](values.yaml) for more de
 | `podTemplate.annotations`                   | Additional annotations to use for the StatefulSet.                                                                                                                                                                                                                                                        | n/a     |
 | `logLevel`                                  | Level to use for logging. Either debug, info, warn, or error.                                                                                                                                                                                                                                             | n/a     |
 | `orgWhiteList`                              | Whitelist of repositories from which Atlantis will accept webhooks. **This value must be set for Atlantis to function correctly.** Accepts wildcard characters (`*`). Multiple values may be comma-separated.                                                                                             | none    |
-| `requireApproval`                           | Whether to require pull request approval prior to applies. See [Approved Requirement](https://www.runatlantis.io/docs/apply-requirements.html#approved).                                                                                                                                                  | `false` |
-| `requireMergeable`                          | Whether to require pull request to be mergeable prior to applies. See [Mergeable Requirement](https://www.runatlantis.io/docs/apply-requirements.html#mergeable).                                                                                                                                         | `false` |
+| `serverSideConfig.enabled`                  | Enable [Server Side Repo Configuration](https://www.runatlantis.io/docs/server-side-repo-config.html)                                                                                                                                                                                                     | `false` |
```

Comment:
can we just detect if there's any data in `serverSideConfig.config`, do we need this boolean? If not, I think we can have a single variable `repoConfig`

Reply:
Makes sense. Changed to `repoConfig` variable only.